public/app/features/dashboard/components/Inspector/QueryInspector.tsx

```diff
@@ -182,8 +215,24 @@ export class QueryInspector extends PureComponent<Props, State> {
     }));
   };
 
+  renderExecutedQueries(executedQueries: ExecutedQueryInfo[]) {
+    const showName = executedQueries.length > 1;
+    return (
+      <div>
+        {executedQueries.map(info => {
+          return (
+            <div key={info.query}>
+              {showName && <h3>{info.name}</h3>}
+              <pre>{info.query}</pre>
+            </div>
+          );
+        })}
+      </div>
+    );
+  }
```

Comment:
nit: moving the length check into this function simplifies the code a bit.
```suggestion
  renderExecutedQueries(executedQueries: ExecutedQueryInfo[]) {
    if(!executedQueries.length) {
      return null;
    }
    return (
      <div>
        {executedQueries.map(info => {
          return (
            <div key={info.query}>
              <h3>{info.name}</h3>
              <pre>{info.query}</pre>
            </div>
          );
        })}
      </div>
    );
  }
```